ledger/accountdb.go

```diff
@@ -2940,6 +3007,54 @@ func accountsPutTotals(tx *sql.Tx, totals ledgercore.AccountTotals, catchpointSt
 	return err
 }
 
+func accountsOnlineRoundParams(tx *sql.Tx) (onlineRoundParamsData []ledgercore.OnlineRoundParamsData, err error) {
+	rows, err := tx.Query("SELECT data FROM onlineroundparamstail")
+	if err != nil {
+		return nil, err
+	}
+	defer rows.Close()
+
+	for rows.Next() {
+		var buf []byte
+		err = rows.Scan(&buf)
+		if err != nil {
+			return nil, err
+		}
+
+		var data ledgercore.OnlineRoundParamsData
+		err = protocol.Decode(buf, &data)
+		if err != nil {
+			return nil, err
+		}
+
+		onlineRoundParamsData = append(onlineRoundParamsData, data)
+	}
+	return
+}
+
+func accountsPutOnlineRoundParams(tx *sql.Tx, onlineRoundParamsData []ledgercore.OnlineRoundParamsData, round basics.Round) error {
+	insertStmt, err := tx.Prepare("REPLACE INTO onlineroundparamstail (round, data) VALUES (?, ?)")
```

Comment:
from review meeting: update to use INSERT instead of REPLACE INTO — unlike the accounttotals table which has just a single row, you are going to have one row per round and should never clobber an existing round's row